corehq/apps/es/tests/test_elastic_sync_multiplexed_command.py

```diff
@@ -204,6 +204,47 @@ def test_delete_index_deletes_the_older_index(self, adapter_patch, input_patch):
         ESSyncUtil().delete_index(HQ_APPS_INDEX_CANONICAL_NAME)
         self.assertFalse(manager.index_exists(app_adapter.index_name))
 
+    def test_remove_residual_indices(self):
+        util = ESSyncUtil()
+
+        # Create known HQ indices if they don't exist
+        existing_index_names = util._get_all_known_indices_name()
```

Comment:
I'm not seeing this method `_get_all_known_indices_name` on updated `master`, nor is it added in this PR? Am I missing something?